cluster-autoscaler/core/static_autoscaler_test.go

```diff
@@ -902,23 +902,23 @@ func TestStaticAutoscalerRunOnceWithFilteringOnBinPackingEstimator(t *testing.T)
 	// Scale up
 	readyNodeLister.SetNodes([]*apiv1.Node{n1, n2})
 	allNodeLister.SetNodes([]*apiv1.Node{n1, n2})
-	scheduledPodMock.On("List").Return([]*apiv1.Pod{p3, p4}, nil).Times(2) // 1 to get pods + 1 when building nodeInfo map
-	unschedulablePodMock.On("List").Return([]*apiv1.Pod{p1}, nil).Once()
+	scheduledPodMock.On("List").Return([]*apiv1.Pod{p3, p4}, nil).Once()
+	scheduledAndUnschedulablePodMock.On("List").Return([]*apiv1.Pod{p3, p4, p1}, nil).Once()
```

Comment:
micro nit: This may be just my OCD, but these pods not being sorted by number gives my chills.

Reply:
Done